pydantic/types.py

```diff
@@ -112,6 +113,7 @@
 
 class ConstrainedBytes(bytes):
     strip_whitespace = False
+    lower_str = False
```

Comment:
```suggestion
    to_lower = False
```

and below.

I think `_str` is tautologous in this context.

`lower` (or `lower_str`) could be confusing, it could be interpreted as meaning the input string must be lowercase, hence I think `to_lower` would be best.